src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -155,5 +170,57 @@ protected override void ConfigureOptions()
         }
 
         #endregion
+
+        public IEnumerable<string> Expand(IEnumerable<string> args)
+        {
+            if (args == null) throw new ArgumentNullException("args");
+
+            foreach (var arg in args)
+            {
+                if (arg.Length == 0 || arg[0] != '@')
+                {
+                    yield return arg;
+                    continue;
+                }
+
+                var fileName = arg.Substring(1, arg.Length - 1);
+                if (string.IsNullOrEmpty(fileName))
+                {
+                    ErrorMessages.Add("The file name should not be empty.");
```

Comment:
This doesn't tell the user much if it's a long command-line... can we say "...after @"?